.github/workflows/release.yml

```
@@ -0,0 +1,96 @@
+name: release
+
+on:
+  release:
+    types: [released]
+  workflow_dispatch:
+
+permissions:
+  contents: read
+  id-token: write
+
+jobs:
+  # Promote and Deploy to prod-test. We do not have api job deploy job for prod-test because we only deploy to worker for prod-test
```

You can still drop promote-prod-test-api below. And this line can be simplified

```suggestion
  # Promote and Deploy to prod-test, which only includes worker
```